parts/k8s/windowsconfigfunc.ps1

```diff
@@ -63,6 +63,11 @@ function Install-Docker
             [System.Environment]::SetEnvironmentVariable('DOCKER_API_VERSION', '1.37', [System.EnvironmentVariableTarget]::Machine)
         }
 
+        "19.03" {
+            Write-Log "Docker 19.03 found, setting DOCKER_API_VERSION to 1.40"
+            [System.Environment]::SetEnvironmentVariable('DOCKER_API_VERSION', '1.40', [System.EnvironmentVariableTarget]::Machine)

```

Comment:
oh my forgot about this workaround for k8s. If you remove this case do the tests fail? I think this was supposed to be a point-in-time workaround for a specific version

Reply:
checking - stay tuned

Reply:
Is there a bug or something that i can reference here for future maintainers?

Reply:
thanks for adding the link. Those two specific versions needed special-case treatment. They could still be in use in some clusters running 1.14 so we shouldn't pull the code out. We just shouldn't set that env var for new deployments using newer Docker versions.